static/app/components/projects/useProjectCreationAccess.tsx

```diff
@@ -1,43 +1,11 @@
-import {useEffect} from 'react';
-
-import {unassignedValue} from 'sentry/data/experimentConfig';
-import {Organization, Team} from 'sentry/types';
-import {useExperiment} from 'sentry/utils/useExperiment';
+import {Organization} from 'sentry/types';
 
 /**
  * Used to determine if viewer can see project creation button
  */
-export function useProjectCreationAccess({
-  organization,
-  teams,
-}: {
-  organization: Organization;
-  teams: Team[];
-}) {
-  const {experimentAssignment, logExperiment} = useExperiment(
-    'ProjectCreationForAllExperimentV2',
-    {
-      logExperimentOnMount: false,
-    }
-  );
-
-  const isAdmin =
-    organization.access.includes('project:admin') ||
-    teams?.some(tm => tm.access.includes('team:admin'));
-
-  const shouldBeInExperiment =
-    !isAdmin &&
-    organization.features.includes('team-project-creation-all') &&
-    experimentAssignment !== unassignedValue;
-
+export function useProjectCreationAccess({organization}: {organization: Organization}) {
   const canCreateProject =
-    isAdmin || (shouldBeInExperiment && experimentAssignment === 1);
-
-  useEffect(() => {
-    if (shouldBeInExperiment) {
-      logExperiment();
-    }
-  }, [logExperiment, shouldBeInExperiment]);
-
+    organization.access.includes('project:admin') ||
+    organization.features.includes('team-roles');
   return {canCreateProject};
```

Comment:
nonblocking q: is there a reason this is an object rather than a boolean?

Reply:
I will merge this today to unblock experimentation removal and fix it tomorrow.

Good point. Before it was an object because this had to be a hook to use experimentation. Now it can just be a util that returns boolean.